drivers/power/lt7170/README.rst

```diff
@@ -0,0 +1,287 @@
+the output voltage to be regulated at the high margin, low margin, or at the
+nominal VOUT_COMMAND. This can also be used to set a channel off.
+
+VIN Configuration
+-----------------
+
+VIN_ON and VIN_OFF command values sets the input voltage window at which power
+conversion will proceed. Both of which can be configured through the
+**lt7170_set_vin** API.
+
+Status Configuration
+--------------------
+
+Statuses bytes/words of the device can be read using **lt7170_read_status**
+API. Faults asserted by these statuses can be cleared using the
+**lt7170_clear_faults** API.
+
+Telemetry
+---------
+
+Measurements can be read using the **lt7170_read_value** API. Some telemetry 
+values includes input/output voltage,
+input/output current, external VCC, die temperature, and output power.
+
+The frequency at which telemetry values are converted, as well as enabling debug
+telemetries such as external VCC and ITH values, can be configured using the
+**lt7170_adc_control** API.
+
+Timing Configuration
+--------------------
+
+The TON_DELAY command sets the time from when a start condition is received
+until the output voltage starts to rise. The TON_RISE sets the time from the
+from the time the output starts to rise to the time the output enters the
+regulation band. The TOFF_DELAY command sets the time from when a stop condition
+is received until the output voltage starts to fall. The TOFF_FALL command sets
+the time from the end of the turn-off delay time until the output voltage is
+commanded to zero. These commands can be set using the **lt7170_set_timing**
+API.
+
+Overvalue and Undervalue Limits Configuration
+---------------------------------------------
+
+Overvalue and undervalue limits sets the threshold at which the device voltage,
+current, temperature, and timing must meet. When these measurements cross the
+limits, a status bit may be asserted. These limits can be configured using the
+**lt7170_set_limit** API.
+
+PWM Configuration
+-----------------
+
+The PWM can be set on forced continuous mode or on
+pulse skip mode. This can be set using the **lt7170_pwm_mode** API.
+
+The phase offset of the PWM can also be set using the
+**lt7170_pwm_phase** API.
+
+Clock Configuration
+-------------------
+
+When using the device in parallel with others of the same device, the PWM clock
+of all devices can be synchronized. Using an external clock or enabling the
+clock input can be configured using the **lt7170_sync_config** API.
+
+NVM commands
+------------
+
+The device has an internal EEPROM that saves the settings configured to the
+device even when power is removed. The **lt7170_nvm_cmd** API allows the user
+to lock and unlock EEPROM, and store, compare and restore user settings.
+
+Software Reset Configuration
+----------------------------
+
+Software Reset operation is available through **lt7170_software_reset** API.
+
+LT7170 Driver Initialization Example
+-------------------------------------
+
+.. code-block:: bash
```

Comment:
Should this be c instead of bash?

Reply:
Used Bash for consistency, as other drivers use Bash instead of C.